packages/api/tests/task.test.js

```diff
@@ -2060,17 +2009,14 @@ describe('Task tests', () => {
         promisedField: [{ location_id }],
       });
 
-      patchTaskDateRequest(
-        { user_id, farm_id },
-        patchTaskDateBody,
-        task.task_id,
-        async (err, res) => {
-          expect(res.status).toBe(200);
-          const updated_task = await getTask(task_id);
-          expect(updated_task.due_date).toBe(due_date);
-          done();
-        },
-      );
+      patchTaskDateRequest({ user_id, farm_id }, patchTaskDateBody, task_id, async (err, res) => {
+        expect(res.status).toBe(200);
+        const updated_task = await getTask(task_id);
+        expect(updated_task.due_date.toISOString().split('T')[0]).toBe(due_date);
```

Comment:
@MwayaB Once I got things running, this test failed. Even though the dates were the same, the value coming back from the database had a timestamp portion to it that was removed from `due_date` when it was initialized. So I removed the timestamp to get them to match.
That feels a bit sloppy, but I think there may be a larger issue behind this. The column in the database stores only the date; I think that the JavaScript/Objection model definition is making it include a (midnight UTC default) timestamp value. This may have to do with recent changes where Jimmy changed some database columns to drop the unused timestamp portions to just be dates.